apps/hash-frontend/src/components/hooks/block-protocol-functions/knowledge/use-block-protocol-file-upload.ts

```diff
@@ -16,6 +17,7 @@ import {
 import { UploadFileRequestCallback } from "./knowledge-shim";
 
 export const useBlockProtocolFileUpload = (
+  ownedById?: OwnedById,
```

Comment:
What is the reason we cannot make `ownedById` required?

Reply:
In various places we call it we don't reliably have the active workspace's account id – e.g. in pages where the viewer might be anonymous (viewing a public page). Ideally we'd rework the logic of pages so that we're not attempting to use this unless we have an authenticated user, but that's a separate piece of work

Reply:
Captured as H-660.